tables/Tables/ArrayColumn.tcc

```diff
@@ -194,17 +194,17 @@ Array<T> ArrayColumn<T>::getSlice (uInt rownr,
 template<class T>
 void ArrayColumn<T>::getSlice (uInt rownr, const Slicer& arraySection,
                                Array<T>& arr, Bool resize) const
-{
+                               {
```

Comment:
Why indented so far? Is it done by your editor or another tool?
In principle current style should be kept.